From e69d8fb1089bd9255488cbe3b9efe8205758d475 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sat, 14 Dec 2019 14:14:32 +0200
Subject: [PATCH 004/230] tuning statusbar looking

	modified:   core/res/res/values/dimens.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_alarm.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_alarm_dim.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_camera.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_data_saver.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_dnd.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_headset.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_headset_mic.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_hotspot.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_location.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_mic_none.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_rotate_landscape.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_rotate_portrait.xml
	modified:   packages/SystemUI/res/drawable/stat_sys_sensors_off.xml
	modified:   packages/SystemUI/res/layout/mobile_signal_group.xml
	modified:   packages/SystemUI/res/layout/status_bar_mobile_signal_group.xml
	modified:   packages/SystemUI/res/layout/status_bar_notification_footer.xml
	modified:   packages/SystemUI/res/layout/status_bar_notification_row.xml
	modified:   packages/SystemUI/res/layout/status_bar_notification_shelf.xml
	modified:   packages/SystemUI/res/layout/status_bar_wifi_group.xml
	modified:   packages/SystemUI/res/layout/system_icons.xml
	modified:   packages/SystemUI/res/values/dimens.xml
---
 core/res/res/values/dimens.xml                |  4 +--
 .../SystemUI/res/drawable/stat_sys_alarm.xml  |  4 +--
 .../res/drawable/stat_sys_alarm_dim.xml       |  4 +--
 .../SystemUI/res/drawable/stat_sys_camera.xml |  6 ++--
 .../res/drawable/stat_sys_data_saver.xml      |  6 ++--
 .../SystemUI/res/drawable/stat_sys_dnd.xml    |  6 ++--
 .../res/drawable/stat_sys_headset.xml         |  4 +--
 .../res/drawable/stat_sys_headset_mic.xml     |  4 +--
 .../res/drawable/stat_sys_hotspot.xml         |  4 +--
 .../res/drawable/stat_sys_location.xml        |  4 +--
 .../res/drawable/stat_sys_mic_none.xml        | 28 +++++++++-------
 .../res/drawable/stat_sys_ringer_silent.xml   |  8 ++---
 .../drawable/stat_sys_rotate_landscape.xml    |  4 +--
 .../res/drawable/stat_sys_rotate_portrait.xml |  4 +--
 .../res/drawable/stat_sys_sensors_off.xml     |  4 +--
 .../res/layout/mobile_signal_group.xml        |  6 ++--
 .../layout/status_bar_mobile_signal_group.xml | 10 +++---
 .../res/layout/status_bar_wifi_group.xml      |  6 ++--
 packages/SystemUI/res/values/dimens.xml       | 32 +++++++++----------
 19 files changed, 76 insertions(+), 72 deletions(-)

diff --git a/core/res/res/values/dimens.xml b/core/res/res/values/dimens.xml
index d165930d3c9..7b518d4cbc1 100644
--- a/core/res/res/values/dimens.xml
+++ b/core/res/res/values/dimens.xml
@@ -74,12 +74,12 @@
     <!-- Width of the navigation bar when it is placed vertically on the screen in car mode -->
     <dimen name="navigation_bar_width_car_mode">96dp</dimen>
     <!-- Height of notification icons in the status bar -->
-    <dimen name="status_bar_icon_size">22dip</dimen>
+    <dimen name="status_bar_icon_size">17dip</dimen>
     <!-- Desired size of system icons in status bar. -->
     <dimen name="status_bar_system_icon_size">15dp</dimen>
     <!-- Intrinsic size of most system icons in status bar. This is the default value that
          is used if a Drawable reports an intrinsic size of 0. -->
-    <dimen name="status_bar_system_icon_intrinsic_size">17dp</dimen>
+    <dimen name="status_bar_system_icon_intrinsic_size">15dp</dimen>
     <!-- Size of the giant number (unread count) in the notifications -->
     <dimen name="status_bar_content_number_size">48sp</dimen>
     <!-- Margin at the edge of the screen to ignore touch events for in the windowshade. -->
diff --git a/packages/SystemUI/res/drawable/stat_sys_alarm.xml b/packages/SystemUI/res/drawable/stat_sys_alarm.xml
index b9bebec5bab..855320cbbc7 100644
--- a/packages/SystemUI/res/drawable/stat_sys_alarm.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_alarm.xml
@@ -16,6 +16,6 @@
 */
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_alarm" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_alarm_dim.xml b/packages/SystemUI/res/drawable/stat_sys_alarm_dim.xml
index cf1119d80fa..364d483707f 100644
--- a/packages/SystemUI/res/drawable/stat_sys_alarm_dim.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_alarm_dim.xml
@@ -14,6 +14,6 @@
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_alarm_dim" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_camera.xml b/packages/SystemUI/res/drawable/stat_sys_camera.xml
index c914262571e..baf3f3206ee 100644
--- a/packages/SystemUI/res/drawable/stat_sys_camera.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_camera.xml
@@ -17,6 +17,6 @@
 */
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="3dp"
-    android:insetRight="3dp"
-    android:drawable="@drawable/ic_camera" />
\ No newline at end of file
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
+    android:drawable="@drawable/ic_camera" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_data_saver.xml b/packages/SystemUI/res/drawable/stat_sys_data_saver.xml
index 022350173a3..98febea2a1a 100644
--- a/packages/SystemUI/res/drawable/stat_sys_data_saver.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_data_saver.xml
@@ -17,6 +17,6 @@
 */
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
-    android:drawable="@drawable/ic_data_saver" />
\ No newline at end of file
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
+    android:drawable="@drawable/ic_data_saver" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_dnd.xml b/packages/SystemUI/res/drawable/stat_sys_dnd.xml
index aa352b42cf0..3a71f086ab1 100644
--- a/packages/SystemUI/res/drawable/stat_sys_dnd.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_dnd.xml
@@ -17,6 +17,6 @@
 */
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
-    android:drawable="@*android:drawable/ic_qs_dnd" />
\ No newline at end of file
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
+    android:drawable="@*android:drawable/ic_qs_dnd" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_headset.xml b/packages/SystemUI/res/drawable/stat_sys_headset.xml
index 361c6659c5e..975b5487b47 100644
--- a/packages/SystemUI/res/drawable/stat_sys_headset.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_headset.xml
@@ -14,6 +14,6 @@
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_headset" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_headset_mic.xml b/packages/SystemUI/res/drawable/stat_sys_headset_mic.xml
index b424caa6e12..790e23af8d4 100644
--- a/packages/SystemUI/res/drawable/stat_sys_headset_mic.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_headset_mic.xml
@@ -14,6 +14,6 @@
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_headset_mic" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_hotspot.xml b/packages/SystemUI/res/drawable/stat_sys_hotspot.xml
index 361b00ae059..1b774910f50 100644
--- a/packages/SystemUI/res/drawable/stat_sys_hotspot.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_hotspot.xml
@@ -14,6 +14,6 @@ Copyright (C) 2017 The Android Open Source Project
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_hotspot" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_location.xml b/packages/SystemUI/res/drawable/stat_sys_location.xml
index 7a5aeb9ae55..fc2c7ae1992 100644
--- a/packages/SystemUI/res/drawable/stat_sys_location.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_location.xml
@@ -14,6 +14,6 @@
   ~ limitations under the License
   -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
     android:drawable="@drawable/ic_location" />
diff --git a/packages/SystemUI/res/drawable/stat_sys_mic_none.xml b/packages/SystemUI/res/drawable/stat_sys_mic_none.xml
index d6bdf9fbaa7..d4ed2c32f26 100644
--- a/packages/SystemUI/res/drawable/stat_sys_mic_none.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_mic_none.xml
@@ -16,15 +16,19 @@
 ** limitations under the License.
 */
 -->
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="18dp"
-    android:height="18dp"
-    android:viewportWidth="24.0"
-    android:viewportHeight="24.0">
-    <path
-        android:fillColor="#FFF"
-        android:pathData="M12,14c1.66,0 3,-1.34 3,-3V5c0,-1.66 -1.34,-3 -3,-3S9,3.34 9,5v6C9,12.66 10.34,14 12,14zM11,5c0,-0.55 0.45,-1 1,-1s1,0.45 1,1v6c0,0.55 -0.45,1 -1,1s-1,-0.45 -1,-1V5z"/>
-    <path
-        android:fillColor="#FFF"
-        android:pathData="M17,11c0,2.76 -2.24,5 -5,5s-5,-2.24 -5,-5H5c0,3.53 2.61,6.43 6,6.92V21h2v-3.08c3.39,-0.49 6,-3.39 6,-6.92H17z"/>
-</vector>
+<inset xmlns:android="http://schemas.android.com/apk/res/android"
+    android:insetLeft="0dp"
+    android:insetRight="0dp">
+    <vector xmlns:android="http://schemas.android.com/apk/res/android"
+      android:width="17dp"
+      android:height="17dp"
+      android:viewportWidth="24.0"
+      android:viewportHeight="24.0">
+      <path
+          android:fillColor="#FFF"
+          android:pathData="M12,14c1.66,0 3,-1.34 3,-3V5c0,-1.66 -1.34,-3 -3,-3S9,3.34 9,5v6C9,12.66 10.34,14 12,14zM11,5c0,-0.55 0.45,-1 1,-1s1,0.45 1,1v6c0,0.55 -0.45,1 -1,1s-1,-0.45 -1,-1V5z"/>
+      <path
+          android:fillColor="#FFF"
+          android:pathData="M17,11c0,2.76 -2.24,5 -5,5s-5,-2.24 -5,-5H5c0,3.53 2.61,6.43 6,6.92V21h2v-3.08c3.39,-0.49 6,-3.39 6,-6.92H17z"/>
+    </vector>
+</inset>
diff --git a/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml b/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
index 1a0599107f6..41a9eebd7ac 100644
--- a/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
@@ -14,10 +14,10 @@ Copyright (C) 2015 The Android Open Source Project
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-        android:insetLeft="0.5dp"
-        android:insetRight="0.5dp">
-    <vector android:width="18dp"
-            android:height="18dp"
+        android:insetLeft="0dp"
+        android:insetRight="0dp">
+    <vector android:width="17dp"
+            android:height="17dp"
             android:viewportWidth="24.0"
             android:viewportHeight="24.0">
         <path
diff --git a/packages/SystemUI/res/drawable/stat_sys_rotate_landscape.xml b/packages/SystemUI/res/drawable/stat_sys_rotate_landscape.xml
index 58ed7e66651..5954b5d0ba6 100644
--- a/packages/SystemUI/res/drawable/stat_sys_rotate_landscape.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_rotate_landscape.xml
@@ -15,8 +15,8 @@
      limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp">
+    android:insetLeft="0dp"
+    android:insetRight="0dp">
     <vector
         android:height="17dp"
         android:width="17dp"
diff --git a/packages/SystemUI/res/drawable/stat_sys_rotate_portrait.xml b/packages/SystemUI/res/drawable/stat_sys_rotate_portrait.xml
index b63ffe0491f..12f2d01c3d1 100644
--- a/packages/SystemUI/res/drawable/stat_sys_rotate_portrait.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_rotate_portrait.xml
@@ -15,8 +15,8 @@
      limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp">
+    android:insetLeft="0dp"
+    android:insetRight="0dp">
     <vector
         android:height="17dp"
         android:width="17dp"
diff --git a/packages/SystemUI/res/drawable/stat_sys_sensors_off.xml b/packages/SystemUI/res/drawable/stat_sys_sensors_off.xml
index ca8f9931f14..a01027f3eaf 100644
--- a/packages/SystemUI/res/drawable/stat_sys_sensors_off.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_sensors_off.xml
@@ -15,8 +15,8 @@
 -->
 
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:insetLeft="3dp"
-        android:insetRight="3dp"
+        android:insetLeft="0dp"
+        android:insetRight="0dp"
         android:width="17dp"
         android:height="17dp"
         android:viewportWidth="24.0"
diff --git a/packages/SystemUI/res/layout/mobile_signal_group.xml b/packages/SystemUI/res/layout/mobile_signal_group.xml
index bfd079b5905..54731b9a127 100644
--- a/packages/SystemUI/res/layout/mobile_signal_group.xml
+++ b/packages/SystemUI/res/layout/mobile_signal_group.xml
@@ -34,14 +34,14 @@
             android:layout_width="wrap_content"
             android:src="@drawable/ic_activity_down"
             android:visibility="gone"
-            android:paddingEnd="2dp"
+            android:paddingEnd="0dp"
             />
         <ImageView
             android:id="@+id/mobile_out"
             android:layout_height="wrap_content"
             android:layout_width="wrap_content"
             android:src="@drawable/ic_activity_up"
-            android:paddingEnd="2dp"
+            android:paddingEnd="0dp"
             android:visibility="gone"
             />
     </FrameLayout>
@@ -51,7 +51,7 @@
         android:layout_width="wrap_content"
         android:layout_gravity="center_vertical"
         android:paddingStart="1dp"
-        android:paddingEnd="2dp"
+        android:paddingEnd="0dp"
         android:visibility="gone" />
     <Space
         android:id="@+id/mobile_roaming_space"
diff --git a/packages/SystemUI/res/layout/status_bar_mobile_signal_group.xml b/packages/SystemUI/res/layout/status_bar_mobile_signal_group.xml
index 42d541e3afc..d4a1a91598e 100644
--- a/packages/SystemUI/res/layout/status_bar_mobile_signal_group.xml
+++ b/packages/SystemUI/res/layout/status_bar_mobile_signal_group.xml
@@ -41,25 +41,25 @@
                 android:layout_height="wrap_content"
                 android:layout_width="wrap_content"
                 android:src="@drawable/ic_activity_down"
+                android:paddingEnd="0dp"
                 android:visibility="gone"
-                android:paddingEnd="2dp"
             />
             <ImageView
                 android:id="@+id/mobile_out"
                 android:layout_height="wrap_content"
                 android:layout_width="wrap_content"
                 android:src="@drawable/ic_activity_up"
-                android:paddingEnd="2dp"
+                android:paddingEnd="0dp"
                 android:visibility="gone"
             />
         </FrameLayout>
         <ImageView
             android:id="@+id/mobile_type"
             android:layout_height="wrap_content"
-            android:layout_width="wrap_content"
+            android:layout_width="12dp"
             android:layout_gravity="center_vertical"
-            android:paddingStart="2.5dp"
-            android:paddingEnd="1dp"
+            android:paddingStart="1dp"
+            android:paddingEnd="0dp"
             android:visibility="gone" />
         <Space
             android:id="@+id/mobile_roaming_space"
diff --git a/packages/SystemUI/res/layout/status_bar_wifi_group.xml b/packages/SystemUI/res/layout/status_bar_wifi_group.xml
index 35cce25d45a..8f9e57199c3 100644
--- a/packages/SystemUI/res/layout/status_bar_wifi_group.xml
+++ b/packages/SystemUI/res/layout/status_bar_wifi_group.xml
@@ -29,7 +29,7 @@
         android:layout_width="wrap_content"
         android:layout_height="match_parent"
         android:gravity="center_vertical"
-        android:layout_marginStart="2.5dp"
+        android:layout_marginStart="0dp"
     >
         <FrameLayout
                 android:id="@+id/inout_container"
@@ -41,15 +41,15 @@
                 android:layout_height="wrap_content"
                 android:layout_width="wrap_content"
                 android:src="@drawable/ic_activity_down"
+                android:paddingEnd="0dp"
                 android:visibility="gone"
-                android:paddingEnd="2dp"
             />
             <ImageView
                 android:id="@+id/wifi_out"
                 android:layout_height="wrap_content"
                 android:layout_width="wrap_content"
                 android:src="@drawable/ic_activity_up"
-                android:paddingEnd="2dp"
+                android:paddingEnd="0dp"
                 android:visibility="gone"
             />
         </FrameLayout>
diff --git a/packages/SystemUI/res/values/dimens.xml b/packages/SystemUI/res/values/dimens.xml
index e414861b0cd..9ec9fd7f32d 100644
--- a/packages/SystemUI/res/values/dimens.xml
+++ b/packages/SystemUI/res/values/dimens.xml
@@ -66,7 +66,7 @@
     <dimen name="status_bar_icon_size">@*android:dimen/status_bar_icon_size</dimen>
 
     <!-- Default horizontal drawable padding for status bar icons. -->
-    <dimen name="status_bar_horizontal_padding">2.5dp</dimen>
+    <dimen name="status_bar_horizontal_padding">0dp</dimen>
 
     <!-- Height of the battery icon in the status bar. -->
     <dimen name="status_bar_battery_icon_height">13.0dp</dimen>
@@ -79,7 +79,7 @@
     <dimen name="status_bar_clock_size">14sp</dimen>
 
     <!-- The starting padding for the clock in the status bar. -->
-    <dimen name="status_bar_clock_starting_padding">7dp</dimen>
+    <dimen name="status_bar_clock_starting_padding">1dp</dimen>
 
     <!-- The end padding for the clock in the status bar. -->
     <dimen name="status_bar_clock_end_padding">0dp</dimen>
@@ -88,16 +88,16 @@
     <dimen name="status_bar_left_clock_starting_padding">0dp</dimen>
 
     <!-- End padding for left-aligned status bar clock -->
-    <dimen name="status_bar_left_clock_end_padding">7dp</dimen>
+    <dimen name="status_bar_left_clock_end_padding">1dp</dimen>
 
     <!-- Spacing after the wifi signals that is present if there are any icons following it. -->
-    <dimen name="status_bar_wifi_signal_spacer_width">2.5dp</dimen>
+    <dimen name="status_bar_wifi_signal_spacer_width">0dp</dimen>
 
     <!-- Size of the view displaying the wifi signal icon in the status bar. -->
     <dimen name="status_bar_wifi_signal_size">@*android:dimen/status_bar_system_icon_size</dimen>
 
     <!-- Spacing before the airplane mode icon if there are any icons preceding it. -->
-    <dimen name="status_bar_airplane_spacer_width">4dp</dimen>
+    <dimen name="status_bar_airplane_spacer_width">0dp</dimen>
 
     <!-- Spacing between system icons. -->
     <dimen name="status_bar_system_icon_spacing">0dp</dimen>
@@ -298,7 +298,7 @@
     <dimen name="overflow_dot_radius">2dp</dimen>
 
     <!-- the padding between dots in the icon overflow -->
-    <dimen name="overflow_icon_dot_padding">3dp</dimen>
+    <dimen name="overflow_icon_dot_padding">0dp</dimen>
 
     <!-- Dimensions related to screenshots -->
 
@@ -766,7 +766,7 @@
     <dimen name="qs_date_time_translation">8dp</dimen>
 
     <!-- Padding before battery level text in status bar, QS, and Keyguard -->
-    <dimen name="battery_level_padding_start">4dp</dimen>
+    <dimen name="battery_level_padding_start">2dp</dimen>
 
     <!-- The top padding of the clear all button -->
     <dimen name="clear_all_padding_top">12dp</dimen>
@@ -787,10 +787,10 @@
     <dimen name="keyguard_carrier_text_margin">16dp</dimen>
 
     <!-- Margin on the left side of the battery % in the header. -->
-    <dimen name="header_battery_margin_expanded">6dp</dimen>
+    <dimen name="header_battery_margin_expanded">3dp</dimen>
 
     <!-- Margin on the left side of the battery % when on Keyguard. -->
-    <dimen name="header_battery_margin_keyguard">6dp</dimen>
+    <dimen name="header_battery_margin_keyguard">3dp</dimen>
 
     <!-- Margin on the right side of the system icon group on Keyguard. -->
     <dimen name="system_icons_keyguard_padding_end">4dp</dimen>
@@ -857,20 +857,20 @@
 
     <!-- Extra padding between the mobile data type icon and the strength indicator when the data
          type icon is wide. -->
-    <dimen name="wide_type_icon_start_padding">2dp</dimen>
+    <dimen name="wide_type_icon_start_padding">0dp</dimen>
 
     <!-- Padding between the mobile signal indicator and the start icon when the roaming icon
          is displayed in the upper left corner. -->
-    <dimen name="roaming_icon_start_padding">2dp</dimen>
+    <dimen name="roaming_icon_start_padding">0dp</dimen>
 
     <!-- Extra padding between multiple phone signal icons. -->
-    <dimen name="secondary_telephony_padding">2dp</dimen>
+    <dimen name="secondary_telephony_padding">0dp</dimen>
 
     <!-- Extra padding between the mobile data type icon and the strength indicator when the data
          type icon is wide for the tile in quick settings. -->
     <dimen name="wide_type_icon_start_padding_qs">3dp</dimen>
 
-    <dimen name="signal_indicator_to_icon_frame_spacing">3dp</dimen>
+    <dimen name="signal_indicator_to_icon_frame_spacing">0dp</dimen>
 
     <!-- The maximum width of the navigation bar ripples. -->
     <dimen name="key_button_ripple_max_width">95dp</dimen>
@@ -882,13 +882,13 @@
     <dimen name="fake_shadow_size">8dp</dimen>
 
     <!-- Starting margin before the signal cluster -->
-    <dimen name="signal_cluster_margin_start">2.5dp</dimen>
+    <dimen name="signal_cluster_margin_start">0dp</dimen>
 
     <!-- Padding between signal cluster and battery icon -->
-    <dimen name="signal_cluster_battery_padding">6dp</dimen>
+    <dimen name="signal_cluster_battery_padding">2dp</dimen>
 
     <!-- Padding for signal cluster and battery icon when there are not icons in signal cluster -->
-    <dimen name="no_signal_cluster_battery_padding">3dp</dimen>
+    <dimen name="no_signal_cluster_battery_padding">2dp</dimen>
 
     <!-- Screen pinning request width -->
     <dimen name="screen_pinning_request_width">@dimen/match_parent</dimen>
-- 
2.25.1

